content/22.vaccines.md

```diff
@@ -406,6 +406,114 @@ Additionally, given that severe SARS-CoV-2 may be associated with a dysregulated
+India has vaccinated approximately 24 million people [@url:https://www.bloomberg.com/graphics/covid-vaccine-tracker-global-distribution/]. 
+This has been achieved by mainly using the AstraZeneca-University of Oxford vaccine, known as Covishield in India, which is also produced by the Serum Institute of India, and using India’s own Covaxin vaccine [@url:https://www.bbc.com/news/world-asia-india-55748124].
+India has also shipped approximately 58 million COVID-19 vaccines to 66 countries [@url:https://www.mea.gov.in/vaccine-supply.htm]
+Considering India produces approximately 60% of the world's vaccines prior to the pandemic, it is no surprise that several other vaccine candidates are under development. 
+These include ZyCov-Di a DNA vaccine produced by Zydus Cadila, HGCO19 India’s first mRNA vaccine produced by Genova and HDT Biotech Corporation (USA), and the Bio E subunit vaccine produced by Biological E in collaboration with US-based Dynavax and the Baylor College of Medicine [@url:https://www.bbc.com/news/world-asia-india-55748124]. 
+
+In China, the Sinopharm-Beijing Institute vaccine, the Sinopharm-Wuhan Institute of Biological Products vaccine, the Sinovac Biotech (CoronaVac) vaccine, and CanSino Biologics vaccine are the main vaccines being distributed. 
+The Sinopharm-Beijing vaccine has been distributed to at least 16 countries. 
+This vaccine is currently approved for use in Bahrain, China, and the United Arab Emirates, but has been granted emergency use in Argentina, Cambodia, Egypt, Guyana, Hungary, Iran, Iraq, Jordan, Nepal, Pakistan, Peru, Venezuela, and Zimbabwe, with limited use in both Serbia and the Seychelles [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html#sinopharm].
+The Sinovac vaccine, CoronaVac, has been approved for use in China, and has been granted emergency use in Azerbaijan, Brazil, Cambodia, Chile, Colombia, Ecuador, Hong Kong, Indonesia, Laos, Malaysia, Mexico, Philippines, Thailand, Turkey, Ukraine, and Uruguay [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html#sinovac].
+Sinovac has reported that their platform now has the capacity to provide up to a billion doses [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html#sinovac].
+Indeed, Sinovac and Sinopharm have estimated that they will be able to produce 2 billion doses by the end of 2021, and they have been able to distribute vaccines as aid to the Philippines and Pakistan [@url:https://www.nytimes.com/2021/01/25/business/china-covid-19-vaccine-backlash.html].
+In contrast, the Sinopharm-Wuhan vaccine, which has been approved for use in China since February 25th, 2021, has been distributed almost exclusively within China, with limited supplies distributed to the United Arab Emirates [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html#wuhan].
```

Comment:
```suggestion
In contrast, the Sinopharm-Wuhan vaccine, which has been approved for use in China since February 25th, 2021, has been distributed almost exclusively within China, with limited supplies distributed to the United Arab Emirates [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html#wuhan].<!--To Do: clean citation-->
```